python/sglang/srt/models/gemma2.py

```diff
@@ -44,6 +44,12 @@
 from sglang.srt.model_executor.forward_batch_info import InputMetadata
 
 
+# Aligned with HF's implementation, using sliding window inclusive with the last token
+# SGLang assumes exclusive
+def get_window_size(config):
```

Comment:
remove this

Reply:
This is for avoiding multiple places of customized computation to reduce the risk of introducing bug in the future.